packages/react-input/src/components/Input/useInputStyles.ts

```diff
@@ -167,15 +172,20 @@ const useRootStyles = makeStyles({
   },
   disabled: {
     cursor: 'not-allowed',
+    backgroundColor: tokens.colorTransparentBackground,
     ...shorthands.border('1px', 'solid', tokens.colorNeutralStrokeDisabled),
     ...shorthands.borderRadius(tokens.borderRadiusMedium), // because underline doesn't usually have a radius
+    '@media (forced-colors: active)': {
+      ...shorthands.borderColor('GrayText'),
```

Comment:
Opinion: I wish the theme would handle this for us rather than having to ensure media queries in every disabled state for every component.

Reply:
@smhigley should confirm but IIRC this would only be needed when we're not using semantic elements (in this case, the wrapper div is visually pretending to be the input)

Reply:
But yeah, I agree that it would be nicer to have this done automatically

Reply:
yeah, it's only for when disabled colors are needed on non-semantic elements, or when we aren't using the native `disabled` attribute (e.g. `disabledFocusable`)